docs/3.x/tech/release-notes/index.adoc

```diff
@@ -12,14 +12,57 @@ Hide Stuff here
 
 // REMEMBER TO ADD MIGRATIONS TO upgrades-patches
 
+[role=release-note]
+== Version 3.31.3
+_Released Tuesday November 9th, 2021_
+
+=== Fixed
```

Comment:
`Github Issue` -> `GitHub Issue`

Reply:
Haha, the exact one I copied from was `Github`.  I fixed that one also.